Updated SharePoint App bar article (#2605)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/sharepoint-app-bar.md

> -	The SharePoint app bar may impact current page customizations specifically those that appear on the left side. 
> -	Personalized content in the SharePoint app bar is enabled by Microsoft Graph.
> -	When Microsoft Graph is disabled, the news and sites experience will be degraded.

> > -	The SharePoint app bar can be temporarily disabled between today and when it becomes available to all customers to give customers more time to prepare for this change. Temporarily disabling the app bar will delay the roll out of this feature in your organization until October 31, 2021.

Reviewer: ```suggestion
> -	The SharePoint app bar can be temporarily disabled between today and when it becomes available to all customers to give customers more time to prepare for this change. Temporarily disabling the app bar will delay the rollout of this feature in your organization until October 31, 2021.
```

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;![Image of where the logo and title will appear in global navigation](media/app-bar-title-logo.png)

> 7.	Finally, determine the **Navigation source**. Learn more about selecting a source below.

Reviewer: ```suggestion
7.	Finally, determine the **Navigation source**. Learn more about selecting a source in the next step.
```

![Image of site navigation](media/app-bar-site.png)

> - Select the site navigation source to display the home site’s navigation.

Reviewer: ```suggestion
- Select the site navigation source to display the home site's navigation.
```

1.	Start by hiding the site navigation using one of two methods:
<br>

> - Go to **Settings**, then **Change the look**, then **Navigation** and toggle the **Display site navigation** to **Hide**

Reviewer: ```suggestion
- Go to **Settings**, then **Change the look**, then **Navigation** and toggle the **Display site navigation** to **Hide**.
```

> - Go to **Settings**, then **Change the look**, then **Header** and choose **Extended** layout

Reviewer: ```suggestion
- Go to **Settings**, then **Change the look**, then **Header** and choose **Extended** layout.
```

3.	[Set this communication](https://docs.microsoft.com/sharepoint/home-site#set-a-site-as-your-home-site) site as a home site using SharePoint Online Management Shell tool.
4.	[Share the home site with users](https://support.microsoft.com/office/share-a-site-958771a8-d041-4eb8-b51c-afea2eae3658?ui=en-us&rs=en-us&ad=us#:~:text=Share%20a%20site%201%20Communication%20sites.%20Select%20Share,external%20users%22%20to%20share%20the%20...%20See%20More.) so they can access the global navigation links. 
5.	On the home site, select **Settings** and then **Global navigation** to **enable** and customize.
6.	For the global navigation source, select **Home site navigation** to display the home site navigation that you created in the global navigation panel, then select **Save**.

## Understand how the app bar may impact page customizations

The SharePoint app bar may impact current page customizations, specifically those positioned to the left of your page. 

> For organization using [page placeholders](https://docs.microsoft.com/sharepoint/dev/spfx/extensions/get-started/using-page-placeholder-with-extensions), the SharePoint app bar will cover parts of both the header and footer page placeholder. In the following image, the placeholder footprint is in red:

Reviewer: ```suggestion
For organizations using [page placeholders](https://docs.microsoft.com/sharepoint/dev/spfx/extensions/get-started/using-page-placeholder-with-extensions), the SharePoint app bar will cover parts of both the header and footer page placeholder. In the following image, the placeholder footprint is in red:
```
